src/services/user/model.js

```diff
@@ -31,8 +36,18 @@ const userSchema = new Schema({
 	timestamps: true,
 });
 
-userSchema.methods.getPermissions = function () {
-	const roleModel = require('../role/model');
+userSchema.virtual('fullName').get(function get() {
+	return [
+		this.namePrefix,
+		this.firstName,
+		this.middleName,
+		this.lastName,
+		this.nameSuffix,
+	].join(' ').trim().replace(/\s+/g, ' ');
```

Comment:
Trimming and replacing double white space here is only necessary, because most of the name fields are optional and we don't want to have a full name like " Peter  Lustig  "